### ISSUE TITLE
can i use it in reasonml?

### ISSUE THREAD

**Issue by useada** [ocaml-ppx/ppx_deriving#178]

**Comment by hcarty** [comment]
Yes, at least for native code compilation. I've used ppx_deriving with Reason, compiled with dune.

**Comment by useada** [comment]
> Yes, at least for native code compilation. I've used ppx_deriving with Reason, compiled with dune.

Great,
thank you very much, I will try this.

**Comment by ELLIOTTCABLE** [comment]
Relevantly, I recently finished fully porting this and publishing it to npm. I intend to maintain this, and keep it up-to-date with the upstream as much as possible.

https://www.npmjs.com/package/bs-deriving#readme

tl;dr of the README:

```sh
$ npm install --save-dev ppx-deriving
$ npm install --save bs-deriving
```

```diff
 "bs-dependencies": [
   ...
+  "bs-deriving"
 ],
+"ppx-flags": [
+   "ppx-deriving/ppx.js"
+],
```

**Comment by whitequark** [comment]
@ELLIOTTCABLE Nice!!

**Comment by spocke** [comment]
@ELLIOTTCABLE Could you update the latest NPM version since the 4.2.1 breaks in the reason language server but that seems to the fixed in the latest 4.4 version but not yet published to npm.

**Comment by kit-ty-kate** [comment]
This looks fixed. Please reopen if not.
